hawkrest/__init__.py

```diff
@@ -61,7 +86,7 @@ def authenticate(self, request):
 
         try:
             receiver = Receiver(
-                lookup_function,
+                lookup_credentials,
```

Comment:
It does -- `lookup_credentials()` is actually defined at the top of `authenticate()` (https://github.com/rneilson/hawkrest/blob/a33b2d786609d1e7c5ea19e910e99a7d4997a6ad/hawkrest/__init__.py#L62) to capture `self` in a closure for passing to `Receiver`. Like I said above, I was trying to keep `_credentials_lookup()` as an instance method, to give maximum flexibility to subclass implementations.